src/filters.js

```diff
@@ -57,6 +57,35 @@ const prefix = (ctx, queryKey, dbKey = queryKey) =>
 const regExp = (ctx, queryKey, dbKey = queryKey, modifiers = 'i') =>
   eq(ctx, queryKey, dbKey, value => new RegExp(escapeRegexp(value), modifiers))
 
+const dateRange = (
+  ctx,
+  queryKeyPrefix,
+  dbKey = queryKeyPrefix,
+  $not = false,
+) => {
+  const dateFrom = ctx.query[`${queryKeyPrefix}From`]
+  const dateTo = ctx.query[`${queryKeyPrefix}To`]
+
+  if (!dateFrom || !dateTo) {
+    return {}
+  }
+
+  let criteria = {
+    $lte: new Date(dateTo),
+    $gte: new Date(dateFrom),
```

Comment:
what if I want to use just the start date? or the end date? it doesnt seem to be taking that option into account

Reply:
then it's not a range 😄 

But I get it, will edit

Reply:
fair enough :D :D but i think its good to be that flexible since it will be in the SDK - for admin purposes